[RFE] Improve error message for provisioning EKS when `desired_size` of node groups is `0`

As part of https://github.com/rancher/terraform-provider-rancher2/issues/989, validation was put in place to prevent the invalid use case of attempting to provision EKS clusters with the `desired_size` of node groups set to `0`.  This issue was opened to request improvements to the current error, which states:
```
[ERROR] can't create %s EKS cluster with node group desired_size = 0.
```

### Describe the solution you'd like
My suggested error message would be as follows:

```
[ERROR] cannot create %s EKS cluster with node group desired_size = 0.  desired_size must be >=1. After initial provisioning, desired_size may be scaled down to 0 at any time.
```


### Additional context

Current error message seen:
<img width="736" alt="Screenshot 2023-07-28 at 4 26 57 PM" src="https://github.com/rancher/terraform-provider-rancher2/assets/46494969/e516e29e-3676-41b1-b4b8-3371c8910590">

--- a/rancher2/resource_rancher2_cluster.go
+++ b/rancher2/resource_rancher2_cluster.go
@@ -149,7 +149,7 @@ func resourceRancher2ClusterCreate(d *schema.ResourceData, meta interface{}) err
 	newCluster := &Cluster{}
 	if cluster.EKSConfig != nil && !cluster.EKSConfig.Imported {
 		if !checkClusterEKSConfigV2NodeGroupsDesiredSize(cluster) {
-			return fmt.Errorf("[ERROR] can't create %s EKS cluster with node group desired_size = 0. desired_size must be >=1. After initial provisioning, desired_size may be scaled down to 0 at any time.", cluster.Name)
+			return fmt.Errorf("[ERROR] can't create %s EKS cluster with node group desired_size = 0. desired_size must be >=1. After initial provisioning, desired_size may be scaled down to 0 at any time", cluster.Name)
 		}
 		clusterStr, _ := interfaceToJSON(cluster)
 		clusterMap, _ := jsonToMapInterface(clusterStr)

--- a/rancher2/resource_rancher2_cluster.go
+++ b/rancher2/resource_rancher2_cluster.go
@@ -149,7 +149,7 @@ func resourceRancher2ClusterCreate(d *schema.ResourceData, meta interface{}) err
 	newCluster := &Cluster{}
 	if cluster.EKSConfig != nil && !cluster.EKSConfig.Imported {
 		if !checkClusterEKSConfigV2NodeGroupsDesiredSize(cluster) {
-			return fmt.Errorf("[ERROR] can't create %s EKS cluster with node group desired_size = 0", cluster.Name)
+			return fmt.Errorf("[ERROR] can't create %s EKS cluster with node group desired_size = 0. desired_size must be >=1. After initial provisioning, desired_size may be scaled down to 0 at any time.", cluster.Name)
 		}
 		clusterStr, _ := interfaceToJSON(cluster)
 		clusterMap, _ := jsonToMapInterface(clusterStr)

